index.js

```diff
@@ -82,32 +83,20 @@ function HttpServer({ listeningPort, rootFolder, sslConfig }, callback) {
 
 		console.log(`Checking if port ${port} is available. Please wait...`);
 
-		require(commType).request({ port }, (res) => {
-			callback(undefined, true);
-		}).on('error', (err) => {
+		const req = require(commType).request({ port }, (res) => {
```

Comment:
The previous implementation had a bug that caused the process to hang if another copy of apihub was running on the same port